src/Swashbuckle.AspNetCore.SwaggerUI/index.html

```diff
@@ -5,113 +5,17 @@
     <meta charset="UTF-8">
     <title>%(DocumentTitle)</title>
     <link rel="stylesheet" type="text/css" href="%(StylesPath)">
+    <link rel="stylesheet" type="text/css" href="./index.css">
     <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
-    <style>
-
-        html {
-            box-sizing: border-box;
-            overflow: -moz-scrollbars-vertical;
-            overflow-y: scroll;
-        }
-
-        *,
-        *:before,
-        *:after {
-            box-sizing: inherit;
-        }
-
-        body {
-            margin: 0;
-            background: #fafafa;
-        }
-    </style>
-    %(HeadContent)
+    %(HeadContent)   
 </head>
 
 <body>
     <div id="swagger-ui"></div>
 
-    <!-- Workaround for https://github.com/swagger-api/swagger-editor/issues/1371 -->
-    <script>
```

Comment:
Is there a possibility that the ordering here was important, so combining the scripts might cause an issue?

Reply:
Edit: if this is still required then the order is essential and I will likely have to go with the extra js file.
Idk when it was fixed on Edge (It's fine in current version - v126.0.2592.87)

Reply:
I guess we can ditch it if it appears to be vestigial and then we can add it back if it turns out it's still a problem for people.

Reply:
Agreed.
Done.